recipes/folly/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set version = "2021.08.02.00" %}  # PEP 386
+{% set number = "0" %}
+
+package:
+  name: folly
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebook/folly/archive/v{{ version }}.tar.gz
+  sha256: 2f9b0ff1af934ca715028462985b6c22004f2599af157bab475857ffc9a5688d
+
+build:
+  number: {{ number }}
+  skip: true  # [not linux64]
+  ignore_run_exports:
+    - jemalloc
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - boost-cpp
+    - double-conversion
+    - fmt
+    - gflags
+    - glog
+    - jemalloc
+    - libaio
+    - libevent
+    - libsodium
+    - snappy
+    - bzip2
+    - lz4-c
+    - openssl
+    - xz
+    - zlib
+    - zstd
+  run:
+    - boost-cpp
+    - double-conversion
+    - fmt
+    - gflags
+    - glog
+    - libaio
+    - libevent
+    - libsodium
+    - snappy
```

Comment:
They should all come in via their `run_exports`

Reply:
Fixed: removed the listed dependencies from `run`